libvcx/src/messages/get_message.rs

```diff
@@ -348,30 +343,22 @@ impl Message {
     fn _decrypt_v3_message(&self) -> VcxResult<::messages::payload::PayloadV1> {
         use v3::messages::a2a::A2AMessage;
         use v3::utils::encryption_envelope::EncryptionEnvelope;
-        use ::issuer_credential::{CredentialOffer, CredentialMessage};
         use ::messages::proofs::proof_message::ProofMessage;
         use ::messages::payload::{PayloadTypes, PayloadV1, PayloadKinds};
         use std::convert::TryInto;
 
         let a2a_message = EncryptionEnvelope::open(self.payload()?)?;
 
         let (kind, msg) = match a2a_message.clone() {
-            A2AMessage::PresentationRequest(presentation_request) => {
-                convert_aries_message!(presentation_request, a2a_message, ProofRequestMessage, ProofRequest)
-            }
-            A2AMessage::CredentialOffer(offer) => {
-                if settings::is_strict_aries_protocol_set() {
-                    (PayloadKinds::CredOffer, json!(&offer).to_string())
-                } else {
-                    let cred_offer: CredentialOffer = offer.try_into()?;
-                    (PayloadKinds::CredOffer, json!(vec![cred_offer]).to_string())
-                }
+            A2AMessage::PresentationRequest(_) => {
+                convert_aries_message!(a2a_message, ProofRequest)
```

Comment:
The macro doesn't seem to provide much utility anymore, I think we can just inline that code?

Reply:
True, will fix